fn_grpc_interface/fn_grpc_interface/util/config.py

```diff
@@ -0,0 +1,27 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+
+"""Generate a default configuration-file section for fn_grpc_interface"""
+
+from __future__ import print_function
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+
+    config_data = u"""[fn_grpc_interface]
+#interface_dir = <<path to interface buffer pb2 files parent directory>>
+#<<package_name=communication_type,secure connection type,certificate_path or google API token>>
```

Comment:
* I still think this needs to be tidier as it does not make sense to me when I read it